server/battles/BattleActionProcessor.cpp

```diff
@@ -1456,14 +1446,13 @@ int64_t BattleActionProcessor::applyBattleEffects(const CBattleInfoCallback & ba
 		CStack::prepareAttacked(bsa, gameHandler->getRandomGenerator(), bai.defender->acquireState()); //calculate casualties
 	}
 
-	int64_t drainedLife = 0;
+	battle::HealInfo tmpHealInfo;
```

Comment:
why do you need this variable? You can write to the passed parameter directly

Reply:
Yeah, it came to me.. just after the commit. Will fix.